clang/test/CIR/CodeGen/inc-dec.cpp

```diff
@@ -0,0 +1,54 @@
+// RUN: %clang_cc1 -triple x86_64-unknown-linux-gnu -fenable-clangir -Wno-unused-value -emit-cir %s -o %t.cir
+// RUN: FileCheck --input-file=%t.cir %s
+
+int id0() {
+    int a = 0;
+    return ++a;
```

Comment:
We usually do 2 spaces for indentation on the tests as well!

Reply:
Indentation in test fixed.